src/components/SearchFacet/SearchFacet.tsx

```diff
@@ -209,9 +235,13 @@ export const SearchFacet = (props: ISearchFacetProps): ReactElement => {
           borderBottomRadius="md"
           mt="0"
         >
-          <FacetStoreProvider facetId={storeId} key={JSON.stringify(searchQuery)}>
-            <FacetList noLoadMore={noLoadMore} onFilter={handleOnFilter} onError={handleOnError} />
-          </FacetStoreProvider>
+          <FacetContext.Provider
```

Comment:
Probably unnecessary, I'd just add these to the current store.  Plus zustand provides us with ability to selective watch properties